apps/api/src/app/integrations/usecases/create-integration/create-integration.usecase.ts

```diff
@@ -69,7 +67,7 @@ export class CreateIntegration {
       result.priority = highestPriorityIntegration ? highestPriorityIntegration.priority + 1 : 1;
     }
 
-    const activeIntegrationsCount = await this.integrationRepository.countActiveExcludingNovu({
+    const activeIntegrationsCount = await this.integrationRepository.countActiveIncludingNovu({
```

Comment:
I'm not sure about this change as it will be out of sync with the UI behavior.
 
The idea before was like that:
1. The user starts by default with Novu integrations, Novu Email is primary.
2. He creates the Mailjet integration.
3. Then, he activates the Mailjet integration. The UI doesn't show the "Select primary provider" modal. It becomes primary as it is the first integration activated excluding Novu integrations.

With the current code in step 3, the integration won't be primary, nor will the "Select primary provider" modal be shown, which is inconsistent with if the user creates another integration and activates it (the modal will be shown).

Reply:
Let's have it like it was before, and we can think about it again.

Reply:
Can you check again?
For me it does shows the select primary modal upon activating mailjet integration.
![image](https://github.com/novuhq/novu/assets/29251776/3cd2344d-b00f-4691-a2cc-2e8ff86e7007)

Reply:
ahh ok, that's why I've added to the DoD this:
```
remove the code on the backend in the create/update integration use-case
- that is counting active integrations and set the primary flag when count is 0
```
ok, it makes sense now if we remove the above code too... 

Like I feel that the user should decide about the primary integration, and we shouldn't do it on his behalf... that's why we will always show the "Select primary provider" if the integration is activated.

WDYT?

Reply:
Yes, in that case I agree with you, it will be better to let users choose
I will remove that part

Reply:
thanks! sorry about the confusion :)